### DISCUSSION TITLE
Is it possible to integrate FSR into other games in this way? If so, how? Is it worth waiting for instructions?

### DISCUSSION THREAD

**Discussion by dkostarnov** [NarutoUA/gta5_fsr#25] [Q&A]
Is it possible to integrate FSR into other games in this way? If so, how? Is it worth waiting for instructions?

**Comment by nickaroot** [comment]
Looks like fsr_common is a key part
https://github.com/miscellaneousrepositories/fsr_common

**Reply by NarutoUA** [reply]
Key part is C++, DirectX, Reverse-Engineering experience.

**Reply by nickaroot** [reply]
Yeah 😃
Key part of fsr*

**Reply by nickaroot** [reply]
Is it true that fsr_common is a universal part?

**Reply by nickaroot** [reply]
We can also reverse d3d part of Cyberpunk 2077 and integrate fsr as a module

**Reply by NarutoUA** [reply]
This is just a some helpers and common stuff I made for myself to not duplicate code in the future if I decide to add fsr somewhere else. If you want to add fsr into any other game you still have to do some reverse-engineering and find out where and how to integrate fsr shaders in that particular game.

**Reply by e5891** [reply]
hey there, i must said that your mod helped me play gta v so much better than ever.
before integrating your mod, usually by set advanced graphics to the fullest / max, even with 0.8 frame scaling i only got 35 - 40 fps, now its 56 - 62 fps with stable performance. 
can you please teach us how to do this magic trick, maybe the "easier way"?
again, thank you so much!

**Comment by tiagomullerbr** [comment]
This mod also seems to work in the Days Gone games (it sure works), Far Cry 5 and Mass Effect Andromeda. I couldn't test at higher resolutions to be sure, as I have an R7 360 and a monitor with a resolution of 1440x900. I believe it makes a bigger difference at higher resolutions. Could anyone test in any of these games?

**Comment by nickaroot** [comment]
Also, we have Cyberpunk 2077 source code (old version)
Maybe it can simplify integration process :)

**Comment by iclook** [comment]
steam-Lossless Scaling
